plugin/trino-pinot/src/test/java/io/trino/plugin/pinot/TestingPinotCluster.java

```diff
@@ -66,8 +66,8 @@
 public class TestingPinotCluster
         implements Closeable
 {
-    public static final String PINOT_LATEST_IMAGE_NAME = "apachepinot/pinot:0.12.1";
-    public static final String PINOT_PREVIOUS_IMAGE_NAME = "apachepinot/pinot:0.11.0";
+    public static final String PINOT_LATEST_IMAGE_NAME = "apachepinot/pinot:1.0.0";
+    public static final String PINOT_PREVIOUS_IMAGE_NAME = "apachepinot/pinot:0.12.1";
```

Comment:
update docs

Reply:
But that just sets the previous version for our tests .. do we basically keep our documented requirements to be the same or is there some real incompatiblity with 0.11.0 .. and if so .. do we have to update the release notes and say that support for Pinot 0.11.0 was removed?

Reply:
We should say it is "not guaranteed". I am reaching out to the Pinot folks to get their opinion: I think we should rely on their compatibility instead of supporting 2 versions as this often leads unexpected issues in one or the other version.
(see linked issue below, just found out and there may be more) and we just document it, ex. Pinot 1.0.1 will be backwards compatible with 1.0.0 but not 0.12.1, etc.
wdyt?

Reply:
I am not sure what our user base is .. so maybe we have to wait a bit and only support 1.0.0 and higher